examples/00-EDB/05_Plot_nets.py

```diff
@@ -48,7 +48,7 @@
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 # Plot all nets on a layer colored by nets.
```

Comment:
```suggestion
# Plot all nets on a layer colored by nets
```
Removing period only because most of these don't have a period. The goal is consistency. It's your choice as to whether to always omit them or always include them.